app-admin/logcheck/logcheck-1.4.3.ebuild

```
@@ -0,0 +1,86 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd readme.gentoo-r1 tmpfiles
```

```suggestion
inherit readme.gentoo-r1 systemd tmpfiles
```

Please sort.

IIRC there is a pkgcheck for this.